code/datums/elements/connect_loc.dm

```diff
@@ -35,7 +35,7 @@
 	if(isnull(old_loc))
 		return
 
-	for (var/signal in connections)
+	for(var/signal in connections)
 		listener.UnregisterSignal(old_loc, signal)
```

Comment:
You can keep the old change here, my post was essentially (It feels wrong) which is no reason to not do something. Honestly shouldn't have made it in the first place, but here we are